tests/phpunit/Integration/JSONScript/TestCases/bibtex-01.json

```diff
@@ -60,6 +60,9 @@
 		{
 			"type": "special",
 			"about": "#0 `format=bibtex` empty (bibtex-01-0.bib)",
+			"skip-on": {
```

Comment:
Do you have the error for this?

Reply:
You mean what is the error for specific test or does this `skip-on` produce an error?

Reply:
I will update the issue, and put the errors so the errors can be visiable here as well.

Reply:
Yup! Thanks!